scripts/pre_commit_linter.py

```diff
@@ -323,14 +309,28 @@ def _pre_commit_linter():
     py_files_to_lint = [
         filename for filename in all_files if filename.endswith('.py')]
 
-    summary_messages = []
-    print '\nStarting jscs linter...'
-    summary_messages.append(_lint_js_files(
-        node_path, jscs_path, config_jscsrc, js_files_to_lint))
-    print '\nStarting pylint...'
-    summary_messages.append(_lint_py_files(
-        config_pylint, py_files_to_lint))
+    results = multiprocessing.Queue()
+    linting_processes = []
+    js_stdout = multiprocessing.Queue()
+    linting_processes.append(multiprocessing.Process(
+        target=_lint_js_files, args=(node_path, jscs_path, config_jscsrc,
+                                     js_files_to_lint, js_stdout, results)))
 
+    linting_processes.append(multiprocessing.Process(
+        target=_lint_py_files, args=(config_pylint, py_files_to_lint, results)))
+    print 'Starting Javascript and Python Linting'
+    print '----------------------------------------'
+    for process in linting_processes:
+        process.start()
+
+    for process in linting_processes:
+        process.join()
+
+    summary_messages = [results.get() for _ in linting_processes]
+    js_messages = [js_stdout.get() for _ in range(js_stdout.qsize())]
```

Comment:
probably better to get() repeatedly until queue is empty. Python docs say that qsize() only gives approx. results.